code/tier_classification/README.md

```diff
@@ -26,7 +26,7 @@ Input file `oncokb_consensus_annotated.txt`
   b. Mutation type: from MAF `Variant_Classification` field - missense, nonsense, indel (frameshift/non-frameshift), splice site, splice region.
   c. Variant allele frequency > 0.05: Calculated as `VAF = t_alt_count/(t_alt_count+t_ref_count)`
   d. Population database: from MAF - `gnomad_AF` < 0.01
-  e. From MAF: `Existing_variation`: Must contain `COSM` identifier (found in COSMIC). 
+  e. From MAF: `Existing_variation`: Must contain `COSM` identifier (found in COSMIC) or is present in COSMIC resistance hotspot table (downloaded from COSMIC)
```

Comment:
Same here - edit to include in point A that non-Tier 1 COSMIC resistance variants are included in Tier 2?

Reply:
+ add that R2 is a criterion given that it is in the code?